spark-jobs/src/test/resources/data/standardization_fixed_width_suite_data.txt

```diff
@@ -1,5 +1,5 @@
 1  Georgina        Richardt
-2  Seamus             Klaes
+ 2 Seamus             Klaes
```

Comment:
```suggestion
2 Seamus             Klaes
```

Reply:
This is on purpose. To show that white space is trimmed both from left and right.

Reply:
ok, my bad